### ISSUE TITLE
global dice suddenly drop to 0 around 700 epoch

### ISSUE THREAD

**Issue by Jines794** [MIC-DKFZ/nnUNet#1518]
Hello,

I am curently training nnUNet on a custom dataset to perform segmentation of radius and ulna on CT scans. so far my dataset is made up of 14 forearm CT-scans and CT-angiography (although the difference isn't specified in each image's name) and their corresponding segmentations, in .nii.gz format. 
As recommended, i ran the command to verify my dataset integrity, and no error was risen. My plan is to run a 5 fold cross-validation with a batch size of 2. 
I started by running the training on fold 0 (without modifying anything on nnUNet except the batch size), and the training seemed to going somewhat  well although the dice was really low. However, around Epoch n°691 my pseudo and global Dice plummeted to 0 , and the train and validation loss, decreased all the way to NaN. I let the training goes continue to Epoch 1000,  there was no recovery and the network architecture wasn't plotted.  
Do you know what is causing this issue and how to solve it ?

Thank you in advance

[training_log_2023_6_22_16_26_49.txt](https://github.com/MIC-DKFZ/nnUNet/files/11869196/training_log_2023_6_22_16_26_49.txt)
[Summary.txt](https://github.com/MIC-DKFZ/nnUNet/files/11869230/Summary.txt)

![image](https://github.com/MIC-DKFZ/nnUNet/assets/98605461/df2d7e1d-bc25-457a-b0bb-482c97cb2b96)
![image](https://github.com/MIC-DKFZ/nnUNet/assets/98605461/37287d80-2be4-4a65-8466-df1b32222af9)
